sci-physics/rivet/rivet-3.1.10.ebuild

```diff
@@ -66,6 +66,7 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}"/${PN}-3.1.6-binreloc.patch
 	"${FILESDIR}"/${PN}-3.1.9-pythontests.patch
+	"${FILESDIR}"/${PN}-3.1.10-ref_cin.patch
```

Comment:
Has this patch been sent upstream? If not, could you do so and add here a link to the MR?

Reply:
https://gitlab.com/hepcedar/rivet/-/merge_requests/844 I just grabbed the patch from https://gitlab.com/hepcedar/rivet/-/merge_requests/844.patch, but it is not obvious from the patch file.